suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -43,7 +43,7 @@ export const groupTransactionsByDate = (
     transactions: WalletAccountTransaction[],
 ): { [key: string]: WalletAccountTransaction[] } => {
     const r: { [key: string]: WalletAccountTransaction[] } = {};
-    const sortedTxs = transactions.sort((a, b) => sortByBlockHeight(a, b));
+    const sortedTxs = [...transactions].sort((a, b) => sortByBlockHeight(a, b));
```

Comment:
Explanation. Need this function to not mutate the original array in order to use it in redux selector. So this is a simple way to "create new array" to make it immutable.

Reply:
Use https://mobily.github.io/ts-belt/api/array#sort instead of this mutable shit :D